actor/builtin/miner/miner.go

```diff
@@ -33,15 +33,19 @@ const MaximumPublicKeySize = 100
 // https://github.com/filecoin-project/go-filecoin/issues/966
 const ProvingPeriodBlocks = 20000
 
+// LargestSectorGenerationAttackThresholdBlocks defines the number of blocks
+// after a proving period ends after which a miner using the largest sector size
+// supported by the network is subject to storage fault slashing.
+//
+// TODO: If the following PR is merged - and the network doesn't define a
+// largest sector size - this constant and consensus.AncestorRoundsNeeded will
```

Comment:
nit: Since you already have a comment in the other PR over consensus.AncestorRoundsNeeded you don't need to mention it here.